### PR TITLE
Integrate nhsuk-frontend

### PR DESCRIPTION
**Pull request by r4vi**

### CODE REVIEW

File: src/client/styles/base.scss [hunk 1]
```diff
@@ -1,5 +1,2 @@
-@import './variables.scss';
-
-body {
-  background-color: $global-bg;
-}
+@import './variables';
+@import "nhsuk-frontend/dist/scss/nhsuk";
```
Comment (tyom) [inline src/client/styles/base.scss]: New line.

File: src/server/plugins/webpack-config/index.js [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
+
     config.plugins.push(new Webpack.HotModuleReplacementPlugin());
+    config.plugins.push(new Webpack.LoaderOptionsPlugin({
+      debug: true
+    }));
+
+    config.plugins = config.plugins.filter(plugin => plugin.constructor.name !== 'ExtractTextPlugin');
+    config.plugins.push(
+      new ExtractTextPlugin({
+        filename: '[name].[hash].css',
```
Comment (tyom) [inline src/server/plugins/webpack-config/index.js]: I wonder if this is required here.

File: src/client/styles/base.scss (hunk 1 above)
Comment (marcofucci) [inline src/client/styles/base.scss]: No empty line at end of the file?

File: src/server/templates/layouts/base.njk [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% extends 'nhsuk-base.njk' %}
+{% block head_stylesheets %}
+  <link rel="stylesheet" href="{{ cssBundle }}" media="screen" type="text/css">
+{% endblock %}
+
+{% block body_footer_content %}
```
Comment (tyom) [inline src/server/templates/layouts/base.njk]:
```jinja
{% block body_footer %}
```

File: src/client/styles/base.scss (hunk 1 above)
Comment (r4vi) [inline src/client/styles/base.scss]: added in https://github.com/nhsuk/register-with-a-gp-beta-web/pull/1/commits/9d28cfab1b48aa61f46bc2e10fa4150e4c108eab

File: src/server/plugins/webpack-config/index.js (hunk 2 above)
Comment (r4vi) [inline src/server/plugins/webpack-config/index.js]: what do you mean? try to mutate the plugin?

**Pull request merged**